CHANGELOG.md

```diff
@@ -0,0 +1 @@
+Set up an environment for the project.
```

Comment:
Something like this format might be nice https://keepachangelog.com/en/1.0.0/

Reply:
Done!